sync-orders-to-bigQuery/.gitignore

```diff
@@ -0,0 +1,37 @@
+# Serverless
+.serverless
+.env
+tmp
+.coveralls.yml
+
+# Google
+keyfile.json
+
+# Logs
+*.log
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
```

Comment:
There's quite a few unnecessary ignores here. This is a generic sls gitignore. The main repos gitignore should ignore most, if not, add the ignore there. 

https://github.com/moltin/integration-examples/blob/master/.gitignore